Disable dynload test. It is known to work but some libtool magic stops it from
working on a system with an already installed gnucash.

--- a/src/gnc-module/test/Makefile.am	2010-12-22 03:01:12.000000000 +0100
+++ b/src/gnc-module/test/Makefile.am	2011-01-25 20:15:21.000000000 +0100
@@ -18,7 +20,6 @@
   test-modsysver \
   test-incompatdep \
   test-agedver \
-  test-dynload \
   test-scm-dynload \
   test-scm-init
 
